UACloudLibraryServer/Controllers/SubmodelRepositoryApi.cs

```diff
@@ -80,8 +81,10 @@ public SubmodelRepositoryApiController(AssetAdministrationShellEnvironmentServic
         [SwaggerResponse(statusCode: 404, type: typeof(Result), description: "Not Found")]
         [SwaggerResponse(statusCode: 500, type: typeof(Result), description: "Internal Server Error")]
         [SwaggerResponse(statusCode: 0, type: typeof(Result), description: "Default error handling for unmentioned status codes")]
-        public async Task<IActionResult> GetAllSubmodelElements([FromRoute][Required] string submodelIdentifier, [FromQuery] int limit, [FromQuery] string cursor, [FromQuery] string level, [FromQuery] string extent, [FromQuery] string diff)
+        public IActionResult GetAllSubmodelElements([FromRoute][Required] string submodelIdentifier, [FromQuery] int limit, [FromQuery] string cursor, [FromQuery] string level, [FromQuery] string extent, [FromQuery] string diff)
         {
+            //CM-Q: Diff is not explained. What is it for?
```

Comment:
https://github.com/admin-shell-io/aas-specs-api is the landing page for the API definition and https://app.swaggerhub.com/apis/Plattform_i40/SubmodelRepositoryServiceSpecification/V3.1.1_SSP-001#/Submodel%20Repository%20API/GetAllSubmodelElements doesn't have it, so please remove.